inst/WORDLIST

```diff
@@ -1,3 +1,130 @@
+️
+andstor
+apache
+args
+aut
+awk
+bioc
+bioccheck
+BiocCheck
+biocViews
+bootswatch
+BugReports
+cd
+Centos
+Changelog
+ci
+CMD
+cobertura
+CODEBASE
+config
+covr
+cre
+cyclocomp
+desc
+DESC
+dev
+dir
+DOCKERFILE
+EndBug
+EnricoMi
+env
+ENV
+expr
+fi
+ganalytics
+gh
+GH
+ghcr
 github
+gitleaks
+GITLEAKS
+grepl
+gsub
+gz
+href
+http
 https
+httr
+ℹ
+insightsengineering
+io
+junit
+JUnit
+knightdave
+knitr
+LazyData
+lewagon
+LibPath
+linkChecker
+linter
+linters
+linux
+logfile
+lychee
+lycheeverse
+md
+multisite
+multiversion
+mv
+navbar
+nestcicd
+NLINES
+nnn
+noreply
+num
+OLDPWD
+oysteR
+packageVersion
+params
+PKGBUILD
+pkgdown
+Pkgdown
+popd
+pre
+pushd
+Rcheck
+readLines
+renv
+repo
+REPO
+repos
+Rmd
+roche
+roclets
+roxygen
+Roxygen
+ROXYGEN
+roxygenize
+RoxygenNote
+Rscript
+rstudio
 RStudio
+sed
+sep
+sessionInfo
+setwd
+shinytest
+softprops
+startsWith
+stefanzweifel
+stringr
+sudo
+SuperLinter
+svenstaro
+testthat
+tidyverse
+tj
+tmp
+tryCatch
+UA
+ubuntu
+usr
+VignetteBuilder
+wget
+www
+xvzf
+yaml
+YAML
+yml
+zricethezav
```

Comment:
A lot of these are from `,github`, so they won't be needed. These are false positives.